backend/schemas/university.py

```
@@ -17,3 +17,9 @@ class UniversityPage(BaseModel):
     total_members: int
     departments: Dict[str, List[Member]]
     post_ids: List[int]
+
+class UniversityListResponse(BaseModel):
+    id: int
+    name: str
+    departments: List[str]
```

Ruff (0.8.2)

Use `list` instead of `List` for type annotation

Replace with `list`

(UP006)